### ISSUE TITLE
Forbid to have anything rather than `Name` in class bases

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#369]
# Rule request

## Thesis

Currently we can do bizarre things with our classes:

```python
class Test((lambda: object)()):  # what?!
     ...
```

We should only allow `ast.Name` to be inside bases classes.

## Reasoning

Classes should not have any complex logic when defined. So, we should limit the scope to `ast.Name` only.

**Comment by nixphix** [comment]
let me work on this

**Comment by sobolevn** [comment]
@nixphix you are welcome. Please, take a note that I will not merge this until we release `0.6.0`.

I would also appreciate your help with #337
